cargo-shuttle/src/lib.rs

```diff
@@ -104,6 +104,21 @@ impl Shuttle {
         self.run(args, provided_path_to_init).await
     }
 
+    fn find_available_port(range_start: u16, range_end: u16) -> u16 {
```

Comment:
Can you add unit tests for this function?

Reply:
sure

Reply:
It seems the tests of `local_run` also need to be updated!

Reply:
may I mark this resolved now?